patch_launch.sh

```
@@ -0,0 +1,30 @@
+sed -i '1596,1624d' setup/launch.py
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)